src/js/DrawStyleController.js

```diff
@@ -56,129 +81,142 @@ goog.require('ga_styles_service');
       ],
       icons: [
         // Basics
-        {id: 'marker', anchor: [0.5, 0.9]},
-        {id: 'circle'},
-        {id: 'square'},
-        {id: 'triangle'},
-        {id: 'star'},
-        {id: 'star-stroked'},
-        {id: 'marker-stroked'},
-        {id: 'circle-stroked'},
-        {id: 'square-stroked'},
-        {id: 'triangle-stroked'},
-        {id: 'cross'},
-        {id: 'disability'},
-        {id: 'danger'},
+        {id: 'marker', anchor: [0.5, 0.9], category: 'standard'},
```

Comment:
The options.icons should be set here.
Move this list of icons in the standard category definition.
Remove also the category property, instead, add it dynamically [here](https://github.com/geoadmin/mf-geoadmin3/pull/4384/files#diff-5fad4113866283bce058fc5c3aaa4bd0R238)